extra/test_release_tool.py

```diff
@@ -13,22 +13,46 @@
 #    limitations under the License.
 
 import os
-import sys
-import shutil
-import re
 import pathlib
+import re
+import shutil
+import sys
 from unittest.mock import patch
 
 import pytest
-
-from release_tool import main
-from release_tool import docker_compose_files_list
-from release_tool import Component
+import yaml
+from release_tool import Component, docker_compose_files_list, main
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 RELEASE_TOOL = os.path.join(THIS_DIR, "release_tool.py")
 INTEGRATION_DIR = os.path.normpath(os.path.join(THIS_DIR, ".."))
 
+# Samples of the different "types" of repos for listing tests
+SAMPLE_REPOS_BACKEND_BASE = ["deviceauth", "gui", "deviceauth", "tenantadm"]
```

Comment:
```suggestion
SAMPLE_REPOS_BACKEND_BASE = ["deviceauth", "gui", "tenantadm"]
```

Reply:
Oh. This probably means that I had some other repo in mind... :thinking:

Reply:
`workflows` maybe?